index.js

```diff
@@ -1,95 +1,226 @@
 
-  heading.textContent = "Create Question";
+  heading.textContent = "Create Card";
 });
 
 profileButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.add("hidden");
-  createSection.classList.add("hidden");
-  profileSection.classList.remove("hidden");
+  hideSection(homeSection);
+  hideSection(bookmarkSection);
+  hideSection(createSection);
+  showSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.remove("navigation-item--active");
-  createIcon.classList.remove("navigation-item--active");
-  profileIcon.classList.add("navigation-item--active");
+  removeHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  addHighlighting(profileIcon);
 
   heading.textContent = "Profile";
 });
 
-/* SET NAVIGATION ACTIVE BUTTON */
+function showSection(section) {
+  section.classList.remove("hidden");
+}
+
+function hideSection(section) {
+  section.classList.add("hidden");
+}
+
+function addHighlighting(icon) {
+  icon.classList.add("navigation-item--active");
+}
+
+function removeHighlighting(icon) {
+  icon.classList.remove("navigation-item--active");
+}
+
+/* CREATE CARD FORM */
+const createCardForm = getElement("createCardForm");
+/* PREVENT DEFAULT*/
+createCardForm.addEventListener("submit", (event) => event.preventDefault());
+
+// CREATE HTML FOR CARD
+let cardContainer = createHTML();
+let cardContainerChildren = cardContainer.children;
+console.log(cardContainerChildren);
+
+// TAKE INPUT VALUES OF FORM
+createCardForm.addEventListener("submit", () => {
+  const createCardQuestion = getElement("createCardQuestion");
+  const createCardAnswer = getElement("createCardAnswer");
+
+  let questionTextInput = createCardQuestion.value;
+  let answerTextInput = createCardAnswer.value;
+
+  // LOOP OVER NODELIST TO REACH HTML ELEMENTS ON NEW CARD
+  for (let i = 0; i < cardContainerChildren.length; i++) {
+    //
+    const questionTextOutput = cardContainerChildren[2];
+    const toggleAnswerButton = cardContainerChildren[3];
+    const answerHeadingOutput = cardContainerChildren[4];
+    const answerTextOutput = cardContainerChildren[5];
+
+    questionTextOutput.textContent = questionTextInput;
+    answerTextOutput.textContent = answerTextInput;
+    // SHOW ANSWER BUTTON OF NEW CARD
+    toggleAnswerButton.addEventListener("click", () => {
+      answerHeadingOutput.classList.toggle("hidden");
+      answerTextOutput.classList.toggle("hidden");
+    });
+    // BOOKMARK TOGGLE OF NEW CARD
+    const toggleBookmarkOutput = cardContainerChildren[0].childNodes[1];
+
+    toggleBookmarkOutput.addEventListener("click", () => {
+      toggleBookmarkOutput.classList.toggle("card__bookmark--active");
+    });
+  }
+
+  // CREATE TAGS ON NEW CARD
+  const createCardTags = getElement("createCardTags");
+  const tagsInputArray = createCardTags.value.split(", ");
```

Comment:
Very nice! To make this _even more robust_, you could split by `","` and the use `trim()` on all strings. That way, it still works if someone forgets a whitespace after a comma.